# HG changeset patch
# User Rik <rik@octave.org>
# Date 1462292928 25200
#      Tue May 03 09:28:48 2016 -0700
# Branch stable
# Node ID 7cb63f3994b0ad5f767cbb7e583184f66a290f96
# Parent  9191c24fb329186585e30272ccc302a084c02c1a
Use correct URL for Online Documentation (bug #47835).

* main-window.cc (open_online_documentation_page): Append index.html to URL.

diff --git a/libgui/src/main-window.cc b/libgui/src/main-window.cc
--- a/libgui/src/main-window.cc
+++ b/libgui/src/main-window.cc
@@ -357,17 +357,18 @@ void
 main_window::handle_new_figure_request (void)
 {
   octave_link::post_event (this, &main_window::new_figure_callback);
 }
 
 void
 main_window::open_online_documentation_page (void)
 {
-  QDesktopServices::openUrl (QUrl ("http://octave.org/doc/interpreter"));
+  QDesktopServices::openUrl (
+    QUrl ("http://octave.org/doc/interpreter/index.html"));
 }
 
 void
 main_window::display_release_notes (void)
 {
   if (! release_notes_window)
     {
       std::string news_file = Voct_etc_dir + "/NEWS";
